src/SonarQube.Client/Api/V7_20/GetIssuesRequest.cs

```diff
@@ -82,7 +82,7 @@ protected override SonarQubeIssue[] ParseResponse(string response)
         /// issues and components, where each issue's "component" property points to a component with
         /// the same "key". We obtain the FilePath of each issue from its corresponding component.
         /// </summary>
-        private ILookup<string, string> componentKeyPathLookup;
+        private protected ILookup<string, string> componentKeyPathLookup;
```

Comment:
Lots of `protected` added. Was it needed?

Reply:
to be able to use these in the derived class to prevent duplication, I thought of maybe separating it but because they will be needed only derived classes it didn't make sense.